pyproject.toml

```diff
@@ -41,6 +43,13 @@ toml = "*"
 lupa = "*"
 fakeredis = "*"
 
+[tool.flake8]
+# B006: Do not use mutable data structures for argument defaults. They are created during function definition time. All calls to the function reuse this one instance of that data structure, persisting changes between them.
+# B008 Do not perform function calls in argument defaults.  The call is performed only once at function definition time. All calls to your function will reuse the result of that definition-time function call.  If this is intended, assign the function call to a module-level variable and use that variable as a default value.
+ignore = "B006,B008"
```

Comment:
[comment]
I would like to ignore these two error codes currently. Maybe fixed in next PR.

Reply:
B006 might be able to avoid potential bugs. Looking forward to consequent PRs :)